packages/state-transition/src/epoch/index.ts

```diff
@@ -65,6 +65,7 @@ const maxSafeValidators = Math.floor(Number.MAX_SAFE_INTEGER / MAX_EFFECTIVE_BAL
 export enum EpochTransitionStep {
   beforeProcessEpoch = "beforeProcessEpoch",
   afterProcessEpoch = "afterProcessEpoch",
+  finalProcessEpoch = "finalProcessEpoch",
```

Comment:
alternative name could be
```suggestion
  endProcessEpoch = "endProcessEpoch",
```
but either way is fine

Reply:
will change if more people want to go with this name, for other names

Reply:
No preference either

Reply:
let's keep as is